Marlin/Configuration.h

```diff
@@ -62,7 +62,7 @@ Here are some standard links for getting your machine calibrated:
 
 // This determines the communication speed of the printer
 // :[2400,9600,19200,38400,57600,115200,250000]
-#define BAUDRATE 250000
+#define BAUDRATE 115200
```

Comment:
This will brake almost all setup guides.
Do you think this is TWI specific? Or do you think this is needed for all?